### PR TITLE
Feature/video

### PR DESCRIPTION
**Pull request by bencullimore**
## Checklist
- [ ] Design and code examples added to the Design System pages
- [ ] Changes added to [CHANGELOG.md](../CHANGELOG.md)
- [ ] Successful linting of the SCSS (`npm run lint-css`)

### CODE REVIEW

**Review by mcheung-nhs — CHANGES_REQUESTED** [review]
Don't think your rebase/merge quite went right...
See:
https://dev-redesign-pr-125.herokuapp.com/styles/images

File: nhsuk-design-system/config/router.js [hunk 1]
```diff
@@ -287,18 +289,6 @@ router.get('/examples/google-allergies', (req, res) => {
   res.render('examples/google-allergies/index', {});
 });
 
-router.get('/examples/test-template-general', (req, res) => {
```
Comment (mcheung-nhs) [inline nhsuk-design-system/config/router.js:290]: Are these meant to have been deleted?

Comment (bencullimore) [reply nhsuk-design-system/config/router.js:290]: no, eek. how do i fix?

Comment (mcheung-nhs) [reply nhsuk-design-system/config/router.js:290]: Probably easiest thing is to see what was deleted here (red background content):
https://github.com/nhsuk/frontend/pull/125/files
and copy it back in to your file, then commit that.

**Review by mcheung-nhs — APPROVED** [review]

**Pull request merged**